src/java/org/apache/cassandra/service/accord/AccordCache.java

```diff
@@ -1190,7 +1190,7 @@ public Runnable save(AccordCommandStore commandStore, TxnId txnId, @Nullable Com
                     return null;
             }
 
-            return commandStore.appendToKeyspace(txnId, value);
+            return null;
```

Comment:
to lower the diff I didn't want to rewrite the whole cache logic... `null` is the only thing possible to return now